numpy_financial/tests/test_financial.py

```diff
@@ -475,3 +443,80 @@ def test_decimal_broadcasting(self):
             Decimal('2000')
         )
         assert_almost_equal(result, desired, decimal=4)
+
+
+class TestFv:
+    def test_float(self):
+        assert_allclose(
+            npf.fv(0.075, 20, -2000, 0, 0),
+            86609.362673042924,
+            rtol=1e-10,
+        )
+
+    def test_decimal(self):
+        assert_almost_equal(
+            npf.fv(Decimal('0.075'), Decimal('20'), Decimal('-2000'), 0, 0),
+            Decimal('86609.36267304300040536731624'),
+            decimal=10,
+        )
+
+    @pytest.mark.parametrize('when', [1, 'begin'])
+    def test_when_is_begin_float(self, when):
+        assert_allclose(
+            npf.fv(0.075, 20, -2000, 0, when),
+            93105.064874,  # Computed using Google Sheet's FV
+            rtol=1e-10,
+        )
+
+    @pytest.mark.parametrize('when', [Decimal('1'), 'begin'])
+    def test_when_is_begin_decimal(self, when):
+        result = npf.fv(
+            Decimal('0.075'),
+            Decimal('20'),
+            Decimal('-2000'),
+            Decimal('0'),
+            when,
+        )
+        assert_almost_equal(result, Decimal('93105.064874'), decimal=5)
+
+    @pytest.mark.parametrize('when', [None, 0, 'end'])
+    def test_when_is_end_float(self, when):
+        args = (0.075, 20, -2000, 0)
+        if when is None:
```

Comment:
nit: Here and below, have you considered converting the following statement to a ternary operator? i.e. `` npf.fv(*args) if when is None else npf.fv(*args, when)``. It is more of a stylistic choice, but, the later reads more easily for me.